lib/kubernetes-deploy/options_helper.rb

```diff
@@ -26,7 +26,7 @@ def default_template_dir(template_dir)
           template_dir = File.join("config", "deploy", ENV['ENVIRONMENT'])
         end
 
-        if !template_dir || template_dir.empty?
```

Comment:
This isn't related, is it? This change just makes the options helper depend on `ActiveSupport`. It's probably always available, but you'd need to tophat all the exes to confirm.

Reply:
New version of rubocop flagged this.

Reply:
Really? That's so weird... `blank` still isn't part of ruby core afaict. Also, it looks like the `lint` step passed on your first two commits on CI.

Reply:
huh, so turns out rubocop is now fine with this file reverted.